app/src/main/java/org/android/go/sopt/IntroduceActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.android.go.sopt
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.android.go.sopt.databinding.ActivityIntroduceBinding
+
+class IntroduceActivity : AppCompatActivity() {
+    private lateinit var viewBinding: ActivityIntroduceBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewBinding = ActivityIntroduceBinding.inflate(layoutInflater)
+        setContentView(viewBinding.root)
+
+        Toast.makeText(
+            applicationContext,
+            "로그인 성공!",
+            Toast.LENGTH_SHORT
+        ).show()
+
+        viewBinding.introduceTvName.text = viewBinding.introduceTvName.text.toString() + intent.getStringExtra("name")
```

Comment:
TextView class 의 append() 를 써주면 더 간단하게 표시될것같네요~

 ```suggestion
        viewBinding.introduceTvName.append(intent.getStringExtra("name"))
```